specs/fault-proof/stage-one/anchor-state-registry.md

```diff
@@ -85,17 +115,89 @@ as blacklisted inside of the `OptimismPortal` contract.
 
 ### Retired Game
 
-A Dispute Game is considered to be a **Retired Game** if the game contract was created before the
-retirement timestamp (`respectedGameTypeUpdatedAt`) defined in the `OptimismPortal` contract.
+A Dispute Game is considered to be a **Retired Game** if the game contract was created with a
+timestamp less than or equal to the retirement timestamp (`respectedGameTypeUpdatedAt`) defined in
+the `OptimismPortal` contract. The retirement timestamp has the effect of retiring all games
+created before the specific transaction in which the retirement timestamp was set. This includes
+all games created in the same block as the transaction that set the retirement timestamp. We
+acknowledge the edge-case that games created in the same block *after* the retirement timestamp
+was set will be considered Retired Games even though they were technically created after the
+retirement timestamp was set.
 
 ### Proper Game
 
-A Dispute Game is considered to be a **Proper Game** if all of the following are true:
+A Dispute Game is considered to be a **Proper Game** if it has not been invalidated through any of
+the mechanisms defined by the `OptimismPortal` contract. A Proper Game is, in a sense, a "clean"
+game that exists in the set of games that are playing out correctly in a bug-free manner. A Dispute
+Game can be a Proper Game even if it has not yet resolved or resolves in favor of the Challenger.
+A Game that was previously a Proper Game can no longer be a Proper Game if it is later invalidated.
+
+Specifically, a game is considered to be a Proper Game if all of the following are true:
+
+- The game is a Registered Game
+- The game is not a Blacklisted Game
+- The game is not a Retired Game
+
+### Resolved Game
+
+A Dispute Game is considered to be a **Resolved Game** if the game has resolved a result in favor
+of either the Challenger or the Defender.
+
+### Airgapped Game
+
+A Dispute Game is considered to be an **Airgapped Game** if the game resolved a result more than
+the airgap delay seconds ago as defined by the `disputeGameFinalityDelaySeconds` variable in the
+`OptimismPortal` contract.
+
+### Finalized Game
+
+A Dispute Game is considered to be a **Finalized Game** if all of the following are true:
+
+- The game is a Resolved Game
+- The game is an Airgapped Game
```

Comment:
This is the same definition as an Airgapped game, because an Airgapped game already requires the game to have resolved.  And "Finalized Game" is a much better term than "Airgapped Game" because all games are airgapped, just some have completed the air gap time and some haven't.

Reply:
Good point, we can get rid of Airgapped and simply keep finalized.